test/cnxml/de/list.js

```diff
@@ -13,6 +13,11 @@ export const input = cnxml`
             <item>Or nested lists</item>
         </list>
     </item>
+    <item>
+        <list>
+            <item>Or mixed text content<para>With paragraph 1</para><para>and paragraph 2</para></item>
```

Comment:
This is incorrect. (and other mixed content elements) can contain _either_ a series of block elements _or_ text, but not both.

Reply:
In some modules there are situations when both of those are present in cnxml. This fix will just handle all possible casese, so we don't have to bother if some content is missing due serialization.